packages/ui/src/2_molecules/Overlay/Overlay.module.css

```
@@ -0,0 +1,47 @@
+.disableScroll {
+  @apply overflow-hidden;
+}
+
+.overlay {
+  @apply inset-0 flex flex-col;
+
+  &.__fixed {
+    @apply fixed;
+  }
+
+  &.__absolute {
+    @apply absolute;
+  }
```

```suggestion
  &.fixed {
    @apply fixed;
  }

  &.absolute {
    @apply absolute;
  }
```
No need to follow `BEM` here.

---

`fixed` and `absolute` are existing default classes on tailwind - will cause error, happy to change to some different name

---

Ah okay, I see that now. In that case it's fine and you can leave it as is :)